src/main/groovy/io/seqera/wave/service/builder/KubeBuildStrategy.groovy

```diff
@@ -117,6 +120,10 @@ class KubeBuildStrategy extends BuildStrategy {
         }
     }
 
+    protected String getBuildImage(BuildRequest buildRequest){
+        return buildRequest.formatDocker() ? kanikoImage : (buildRequest.platform.arch == "arm64"?singularityImageArm:singularityImage)
+    }
```

Comment:
Let's add a unit test for this. Also some formatting can make it more readable

Reply:
Unit Tests are already added

Reply:
I was missing that 👍